src/datacamp_downloader/datacamp_utils.py

```diff
@@ -288,44 +288,46 @@ def download_others(self, course_id, chapter: Chapter, path: Path, **kwargs):
                 )
                 exercise_counter += 1
             if exercise.is_video:
-                video = self._get_video(exercise.data.get("projector_key"))
-                video_path = path / "videos" / f"ch{chapter.number}_{video_counter}"
-                if videos and video.video_mp4_link:
-                    download_file(
-                        self.session,
-                        video.video_mp4_link,
-                        video_path.with_suffix(".mp4"),
-                        overwrite=self.overwrite,
-                    )
-                if audios and video.audio_link:
-                    download_file(
-                        self.session,
-                        video.audio_link,
-                        path / "audios" / f"ch{chapter.number}_{video_counter}.mp3",
-                        False,
-                        overwrite=self.overwrite,
-                    )
-                if scripts and video.script_link:
-                    download_file(
-                        self.session,
-                        video.script_link,
-                        path / "scripts" / (video_path.name + "_script.md"),
-                        False,
-                        overwrite=self.overwrite,
-                    )
-                if subtitles and video.subtitles:
-                    for sub in subtitles:
-                        subtitle = self._get_subtitle(sub, video)
-                        if not subtitle:
-                            continue
+                projector_key = exercise.data.get("projector_key")
+                if projector_key is not None:
+                    video = self._get_video(projector_key)
+                    video_path = path / "videos" / f"ch{chapter.number}_{video_counter}"
+                    if videos and video.video_mp4_link:
+                        download_file(
+                            self.session,
+                            video.video_mp4_link,
+                            video_path.with_suffix(".mp4"),
+                            overwrite=self.overwrite,
+                        )
+                    if audios and video.audio_link:
+                        download_file(
+                            self.session,
+                            video.audio_link,
+                            path / "audios" / f"ch{chapter.number}_{video_counter}.mp3",
+                            False,
+                            overwrite=self.overwrite,
+                        )
+                    if scripts and video.script_link:
```

Comment:
For cleaner code, you should revert your changes and add only the following:
`
if not video:
    continue
`